cfgov/templates/jobmanager/supervision.html

```diff
@@ -13,7 +17,7 @@
 {% block breadcrumbs %}
 	<li><a href="/" class="internal-link">Home</a></li>
```

Comment:
Might as well match the indent of the lines below

Reply:
Looking at some of the other ones, I removed the indents in the block, because they're not used elsewhere. Has been updated!